test/stackAnalysisService.test.ts

```diff
@@ -112,20 +113,21 @@ suite('stacknalysis Services', () => {
     let stubRequestPost = sandbox
       .stub(request, 'post')
       .yields(null, { statusCode: 500 });
-    stackAnalysisServices.postStackAnalysisService(options, context);
+    stackAnalysisServices.postStackAnalysisService(options, context, retryCount);
     expect(stubRequestPost).callCount(1);
     expect(spyClearContextInfo).callCount(1);
   });
 
   test('postStackAnalysisService should return error', () => {
+    let retryCount = 999;
```

Comment:
Why 999 retries?

Reply:
the goal in this test is to skip the retry mechanism and only test the functionality itself. 
in order to surpass any logical MAX_RETRIES value, so this statement `else if (retryCount < MAX_RETRIES)` will return false, retry must be a very high/unrealistic number. 
so i set it to 999

Reply:
I agree, thanks for the catch!
Ill take care of it using the Numbers.MAX_VALUE property